interface/src/app/download/download.js

```diff
@@ -1,80 +1,74 @@
 /**
- * "download" module, which will download sample annotations of given
- * annotation typenames.
+ * "adage.download" module, which will download sample annotations based on
+ * selected annotation types.
  */
 
 angular.module('adage.download', [
   'ui.router',
-  'placeholders',
   'ui.bootstrap',
+  'ngResource',
   'as.sortable'
 ])
 
-.config(function config($stateProvider) {
+.config(['$stateProvider', function config($stateProvider) {
   $stateProvider.state('download', {
     url: '/download',
     views: {
-      'main': {
-        controller: 'DownloadCtrl',
-        templateUrl: 'download/download.tpl.html'
+      main: {
+        templateUrl: 'download/download.tpl.html',
+        controller: 'DownloadCtrl as ctrl'
       }
     },
     data: {pageTitle: 'Download'}
   });
-})
+}])
 
-// "AnnotationTypes" is a service that retrieves a list of available
-// annotation columns, from which the user can select for download.
 .factory('AnnotationTypes', ['$resource', function($resource) {
-  return $resource(
-    '/api/v0/annotationtype/',
-    {q: '', limit: 0},
-    {'query': {method: 'GET', isArray: false}}
-  );
+  return $resource('/api/v0/annotationtype');
 }])
 
-.controller('DownloadCtrl', ['$scope', '$window', 'AnnotationTypes',
-  function DownloadController($scope, $window, AnnotationTypes) {
-    $scope.annotations = {
-      query_message: 'Connecting to the server ...',
-      included_types: [],
-      excluded_types: []
-    };
-
-    AnnotationTypes.query(
-      {q: ''},
-      function(responseObject, responseHeaders) {
-        for (var i = 0; i < responseObject.objects.length; ++i) {
-          $scope.annotations.included_types.push(
-            responseObject.objects[i].typename);
-        }
-        $scope.annotations.query_message = '';
-      },
-      function(responseObject, responseHeaders) {
-        $log.error('Query errored with: ' + responseObject);
-        $scope.annotations.query_message = 'Connection to server failed';
-      }
-    );
+.controller('DownloadCtrl', ['$window', '$log', 'AnnotationTypes',
+  function DownloadController($window, $log, AnnotationTypes) {
+    var self = this;
```

Comment:
I didn't know that **window.self** already exists.
From what I read, **self** seems to be a variable name used widely in JavaScript community. Google's AngularJS style guide recommends **controllerAs** syntax, but it doesn't specify whether it is a good idea to assign `this` to another variable:
https://google.github.io/styleguide/angularjs-google-style.html#dollarsign
(The example is too simple to assign `this` to another variable.)

Another style guide that seems to be popular actually recommend using the variable name **vm** (which stands for "view model") to refer to the controller instance:
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#controlleras-with-vm 
As you said, this is totally a style issue. Once we reach agreement, whatever name is okay with me.